data/sql/updates/pending_db_world/rev_1622194253034538200.sql

```diff
@@ -0,0 +1,7052 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1622194253034538200');
+
+ALTER TABLE `creature_template` ADD COLUMN `detection_range` FLOAT NOT NULL DEFAULT 20 AFTER `speed_run`;
+
+UPDATE creature_template SET detection_range ='1' WHERE name ="Baron Rivendare";
```

Comment:
https://www.azerothcore.org/wiki/sql-standards lot of things to correct :D

Reply:
I'll redo it on the entry NPCs, but I won't be responsible if you change them :-D

Reply:
Just dont forget to use backticks and not to use single quotes on integers